Formula/rubberband.rb

```diff
@@ -4,7 +4,7 @@ class Rubberband < Formula
   head "https://bitbucket.org/breakfastquay/rubberband/", :using => :hg
 
   stable do
-    url "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2"
```

Comment:
This seems to be the canonical url.

Reply:
Oh I see the download is missing. Probably should let them know:

https://bitbucket.org/breakfastquay/rubberband/issues

Reply:
It's the download link from the homepage, but the 'files' area seems to be misconfigured and I couldn't find it anywhere else either. The link also looks on the "accidental" side to be worthy noting in a comment and easy to lookup on the homepage, so I haven't made a copy-paste of it.

Reply:
Upstream report: https://bitbucket.org/breakfastquay/rubberband/issues/21/